recipe/meta.yaml

```
@@ -1,125 +1,85 @@
+    # We also need the "blas" package. Otherwise we get reports
+    # of "cblas.h: No such file or directory" at import time.
+    # Ref: <https://github.com/conda-forge/aesara-feedstock/issues/80>
+    - blas
+    - wheel
+  run:
+    - python
+    - {{ pin_compatible('numpy') }}
+    - scipy >=0.14
+    - filelock
+    - etuples
+    - logical-unification
+    - miniKanren
+    - cons
+    - typing_extensions
+    - setuptools >=48.0.0
 
-  - name: pytensor
-    build:
-      script:
-        - echo "Nothing to build here, just add dependencies."
-    requirements:
-      build:
-        - {{ compiler('c') }}
-        - {{ compiler('cxx') }}
-      host:
-        - python
-      run:
-        - python
-        - {{ pin_subpackage('pytensor-base', exact=True) }}
-        - {{ compiler('c') }}  # [not win]
-        - {{ compiler('cxx') }}  # [not win]
-        - gxx  # [linux]
-        - {{ compiler('m2w64_c') }}  # [win]
-        - {{ compiler('m2w64_cxx') }}  # [win]
-
-        - m2w64-toolchain  # [win]
-        # The libpython conda-forge package seems to be a wrapper package
-        # which configures m2w64-toolchain for Python.
-        # <https://github.com/conda-forge/libpython-feedstock/blob/main/recipe/bld.bat>
-        - libpython >=2.2  # [win]
-
-        # mkl-service is only available for linux-64, win-64, and osx-64.
-        # For everything else, use openblas.
-        - mkl-service  # [linux64 or win64 or (osx and not arm64)]
-        - openblas  # [not (linux64 or win64 or (osx and not arm64))]
-        # For some reason we also need the "blas" package. Otherwise we get reports
-        # of "cblas.h: No such file or directory" at import time.
-        # Ref: <https://github.com/conda-forge/aesara-feedstock/issues/80>
-        - blas
-    test:
-      imports:
-        - pytensor
-        - pytensor.configdefaults
-      commands:
-        - pytensor-cache help
-        - pip check
-        - python check-for-warnings.py allowed-warnings-main.txt
-      requires:
-        - pip
-      files:
-        - check-for-warnings.py
-        - allowed-warnings-main.txt
+test:
+  imports:
+    - pytensor
+    - pytensor.configdefaults
+  requires:
+    - pip
+    - pytest
+    - pre-commit
+    - pytest-cov >=2.6.1
+    - coverage >=5.1
+    - pytest-benchmark
+  source_files:
+    - tests
+  files:
+    - check-for-warnings.py
+    - allowed-warnings-base.txt
+  commands:
+    - pytensor-cache help
+    - pip check
+    - python check-for-warnings.py allowed-warnings-base.txt
+    - python -m pytest tests/test_breakpoint.py tests/test_config.py tests/test_gradient.py tests/test_ifelse.py tests/test_raise_op.py tests/test_rop.py tests/test_updates.py
+    - python -m pytest tests/test_printing.py  # [osx]
 
 about:
   home: https://github.com/pymc-devs/pytensor
   license: BSD-3-Clause
-  summary: An optimizing compiler for evaluating mathematical expressions.
+  license_family: BSD
   license_file: LICENSE.txt
```

maybe you can fix the license file problem by changing this to doc/LICENSE.txt instead of bundling the file to the feedstock
```suggestion
  license_file: doc/LICENSE.txt
```

---

let me give this a try!